capa/features/extractors/ida/insn.py

```diff
@@ -35,7 +37,7 @@ def get_externs(ctx: Dict[str, Any]) -> Dict[int, Any]:
     return ctx["externs_cache"]
 
 
-def check_for_api_call(insn: idaapi.insn_t, funcs: Dict[int, Any]) -> Iterator[Any]:
+def check_for_api_call(insn: idaapi.insn_t, funcs: Dict[int, Any]) -> Optional[Tuple[str, str]]:
```

Comment:
mypy complains for this function

Reply:
maybe try to type the tuple initialization? alternatively, add a noqa or similar comment?